src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -585,6 +618,42 @@ def _resolve_time_spent_percentage(
             alias,
         )
 
+    def _resolve_http_response_code_count(
+        self,
+        args: Mapping[str, str | Column | SelectType | int | float],
+        alias: str | None = None,
+        extra_condition: Function | None = None,
```

Comment:
should just remove the extra_condition param

Reply:
Done!